providers/bnxt_re/main.h

```diff
@@ -554,6 +557,11 @@ static inline void bnxt_re_jqq_mod_last(struct bnxt_re_joint_queue *jqq,
 	jqq->last_idx = jqq->swque[idx].next_idx;
 }
 
+static inline uint32_t bnxt_re_get_diff(uint64_t cmask)
```

Comment:
1. It is not used
2. There is no value in function which simply returns defind

Reply:
Agree that it is not useful with the current patch. I will remove it.
I have a follow up series which avoid this diff for newer adapters, that includes kernel driver changes. We will have to check the compatibility mask for adding the diff. I will re-introduce the function for that series.